hack/verify-dependencies.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
If this is to be long-lived, I need a _strong_ argument to convince me that it should be written in Bash.